build.sh

```diff
@@ -930,30 +946,54 @@ if [[ "${ISSUE_CLEAN_AGGRESSIVE}" == "true" ]]; then
     build_clean_aggressive
 fi
 
-if [[ "${ISSUE_DESKTOP_BUILD}" == "true" ]]; then
```

Comment:
Why did you reorder the tests?

Reply:
I checked, and reordering the `builds` doesn't make sense. reverted.